src/main/java/Torne.java

```diff
@@ -0,0 +1,398 @@
+import java.util.*;
+
+/**
+ * This is the main class for `Torne`.
+ * All the main commands are here.
+ */
+
+public class Torne {
+    private static final ChatOutput OUTPUT = new ChatOutput();
+    private static TaskHandler taskHandler;
+    private static Storage storage;
+    private static final String[] NO_ARGS = new String[0];
+    private static final String[] DEFAULT_ARG = {""};
+    private static final Map<String, String[]> COMMANDS = Map.of(
```

Comment:
Awesome job in creating a Map that effectively reduces the number of classes you have to make (Command class etc)